src/ludamus/templates/chronology/event.html

```
@@ -432,15 +432,18 @@ <h2 class="text-xl font-bold text-foreground">
                             </div>
                         </div>
                     </div>
-                    {% if compact_schedule or has_enrollable_sessions %}
+                    {# The switcher exists for the rooms grid, which renders from #}
+                    {# schedule_days — only the compact layout builds those, so a #}
+                    {# card-grid event has a single view and no control to offer. #}
+                    {% if compact_schedule %}
                         {# hx-boost rather than hx-get on each tab: {% tab %} renders a #}
                         {# plain <a> and passes no attribute through, and the boost stays #}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a> ] on line 440.

(tag-pair)